docs/docs/gatsby-cli.md

```diff
@@ -66,121 +66,121 @@ gatsby new
    (Use a different starter)
 ```
 
-See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/) for more details.
+अधिक विवरण के लिए [Gatsby शुरुआत डॉक्स](https://www.gatsbyjs.org/docs/gatsby-starters/) देखें।
 
-### `develop`
+### `विकसित करना`
 
-Once you've installed a Gatsby site, go to the root directory of your project and start the development server:
+एक बार जब आप एक Gatsby साइट इनस्टॉल कर लेते हैं, तो अपनी प्रॉजेक्ट की रूट फोल्डर मे जाएँ और डेवेलपमेंट सर्वर शुरू करें:
 
 `gatsby develop`
 
-#### Options
+#### विकल्प
 
-|     Option      | Description                                     |
+|     विकल्प      | विवरण                                     |
 | :-------------: | ----------------------------------------------- |
-| `-H`, `--host`  | Set host. Defaults to localhost                 |
-| `-p`, `--port`  | Set port. Defaults to 8000                      |
-| `-o`, `--open`  | Open the site in your (default) browser for you |
-| `-S`, `--https` | Use HTTPS                                       |
+ `-H`, `--host`  | होस्ट सेट करें। डिफॉल्ट लोकलहोस्ट             |
```

Comment:
```
`-H`, `--host`  | होस्ट सेट करें। डिफॉल्ट लोकलहोस्ट             |
```
=>
```
|`-H`, `--host` | होस्ट सेट करें। इसकी डिफ़ॉल्ट वैल्यू localhost है               |
```

How about this? I think this makes it more clear to understand. Any thoughts?
Copy-paste the complete line, so alignment will not break

Reply:
yes...

Reply:
This you have missed, I guess. Can you check it again

Reply:
I think, You are referring outdated code. Please check it again.

Reply:
This is not updated. Check the attachment, you are missing `|` at start

<img width="675" alt="Screen Shot 2019-11-13 at 11 57 55 AM" src="https://user-images.githubusercontent.com/4654382/68738661-e768ca00-060c-11ea-8c26-0c6cfba342ee.png">

Reply:
my bad. fixed now